app/forms/hyrax/una_chapters_and_book_section_form.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+module Hyrax
+  class UnaChaptersAndBookSectionForm < Hyrax::Forms::WorkForm
+    include ::HykuAddons::WorkForm
+
+    self.model_class = ::UnaChaptersAndBookSection
```

Comment:
Can you try and format these sections the same. This file has spaces before the `include` and not after the `add_terms` like `UnaBookForm`. There should not be any differences in how classes are structured or how styling is applied.

Reply:
They should be the same since I just duplicate an old file but this was duplicated from a Redlands worktype so might be slightly different

Reply:
Ill make them consistent